--- a/include/linux/sched.h
+++ b/include/linux/sched.h
@@ -1000,15 +1000,15 @@ struct task_struct {
 #endif
 
 #ifdef CONFIG_SYSVIPC
-	struct sysv_sem		sysvsem;
-	struct sysv_shm		sysvshm;
+	ANDROID_KABI_USE(2, struct sysv_sem sysvsem);
+	_ANDROID_KABI_REPLACE(3, 4, struct sysv_shm sysvshm);
 #endif
 #ifdef CONFIG_DETECT_HUNG_TASK
-	unsigned long		last_switch_count;
-	unsigned long		last_switch_time;
+	ANDROID_KABI_USE(5, unsigned long last_switch_count);
+	ANDROID_KABI_USE(6, unsigned long last_switch_time);
 #endif
 #ifdef CONFIG_IO_URING
-	struct io_uring_task	*io_uring;
+	ANDROID_KABI_USE(7, struct io_uring_task *io_uring);
 #endif
 
 	/*
